ports/mimxrt10xx/background.c

```diff
@@ -27,7 +27,11 @@
 
 #include "supervisor/port.h"
 
-void port_background_task(void) {
+#include "supervisor/linker.h"
+
+#include "fsl_common.h"
+
+void PLACE_IN_ITCM(port_background_task)(void) {
 }
```

Comment:
we should consider supplying a weak version of this, noting that it should be marked with this macro across all ports as well.